demo/src/routes/about.astro

```diff
@@ -1,10 +1,15 @@
 ---
 import { t } from "i18n:astro";
 import Layout from "~/layouts/Layout.astro";
+import HeroMessage from "~/components/HeroMessage.astro";
 
 const title = t("about");
 ---
 
 <Layout {title}>
-  <h1>{title}</h1>
+  <div class="justify-center items-center">
+    <HeroMessage message="aboutMessage" />
+    <HeroMessage message="opensource" />
+    <a href="https://github.com/astrolicious/i18n/tree/main/demo" class="underline"><HeroMessage message="opensourceCTA" /></a>
```

Comment:
Don't remember if it's there already but can you add a link in the header as well?

Reply:
Sure!